src/index.ts

```diff
@@ -63,6 +64,7 @@ export default class IronfishApp extends GenericApp {
         DKG_BACKUP_KEYS: 0x19,
         DKG_RESTORE_KEYS: 0x1a,
         GET_RESULT: 0x1b,
+        REVIEW_TX: 0x1c,
```

Comment:
I think we have to add this in types.ts too

Reply:
The definition is missed here 
https://github.com/Zondax/ledger-ironfish-js/blob/a9857eac0be6276d633c783231281e1e0e25de33/src/types.ts#L6C1-L18C2

Reply:
did you refresh?
that change is listed in the files changed menu